protocol/0026-AUCT-auctions.md

```diff
@@ -177,4 +176,4 @@ message Market {
 - As an API user, I can identify: (<a name="0026-AUCT-008" href="#0026-AUCT-008">0026-AUCT-008</a>)
   - If a market is temporarily in an auction period
   - Why it is in that period (e.g. Auction at open, liquidity sourcing)
-  - When the auction mode ends
+  - When the auction will next attempt to uncross and end
```

Comment:
Note for pedants: this can be *right this second* in the case of an extending liquidity auction, in which case it's probably in the past, which looks odd.